### DISCUSSION TITLE
is anyone using text to speech? if so, which?

### DISCUSSION THREAD

**Discussion by joshgura** [LostRuins/koboldcpp#1119] [Q&A] [answered]
my knowledge level is almost nothing about TTS, but I would like to try. XTTS wasn't working and I'm about to try Alltalk I guess. 

anyone, especially linux users using TTS or a combination of whisper? Kobold is sending text to the xtts server but I'm hitting errors on that end. I thought it would be a lot of fun to play with Coqui too, but couldn't get it to even boot. 

i see the words "whisper" and "TTS" in kobold and I'm like "awesome!" but the reality of random third party stuff breaking and becoming obsolete every other week isn't so awesome. 

help? I'd be glad to get _any_ tts to work for starters.

**Comment by Abdulhanan535** [comment, marked as answer]
- XTTSv2
- AllTalk (Has XTTSV2 incl.)
- ElevenLabs
- RVC (Real Voice Cloning)
